client/my-sites/themes/themes-magic-search-card/index.jsx

```diff
@@ -295,7 +295,7 @@ class ThemesMagicSearchCard extends React.Component {
 								/>
 							</div>
 						}
-						{ isPremiumThemesEnabled && ! isJetpack &&
+						{ config.isEnabled( 'jetpack/pijp' ) && isPremiumThemesEnabled &&
```

Comment:
If config is disabled, it should check the previous conditional `isPremiumThemesEnabled && ! isJetpack`